### ISSUE TITLE
Docker container doesn't work with sqlite as database

### ISSUE THREAD

**Issue by gloriousDan** [TandoorRecipes/recipes#1614]
### Tandoor Version

v1.1.0

### Setup

Docker / Docker-Compose

### Reverse Proxy

Traefik

### Bug description

I'm running tandoor together with a sqlilte database in conjunction with litestream (https://litestream.io/) to back it up. 

My env file looks like this:
```env
# add only a database password if you want to run with the default postgres, otherwise change settings accordingly
DB_ENGINE=django.db.backends.sqlite3
# DB_OPTIONS= {} # e.g. {"sslmode":"require"} to enable ssl
#POSTGRES_HOST=db_recipes
#POSTGRES_PORT=5432
#POSTGRES_USER=djangouser
#POSTGRES_PASSWORD=""
# this is where I mount the sqlite litestream db
POSTGRES_DB=/opt/recipes/db/db.sqlite3
```

With the changes in v1.1.0 to `boot.sh` the tandoor container doesn't start anymore.

Specifically the checks in line 24 - 35 seem to be causing problems, since the tandoor container won't start until a postges db is detected as running and when running sqlite no such db exists:
https://github.com/TandoorRecipes/recipes/blob/f9f6ca05a4a03b0ad2fb20abac507782b23fbd41/boot.sh#L24-L35

I can try to look at how sqlite is used in the dev setup and try to implement a fix/ exception for sqlite myself.

**Comment by MaxJa4** [comment]
Since most don't use SQLite with Tandoor, I haven't taken this into account

I can implement a check so the PSQL related stuff is only executed when the DB PSQL is actually used.

@vabene1111 Since it's probably just one little if statement... do you want to include this into the 1.0.1 (I suppose the arm release with the build fix will be the 1.0.1?)?

**Comment by gloriousDan** [comment]
The check for POSTGRES_PASSWORD also failed since I didn't specify the POSTGRES_PASSWORD but since that's only a warning it doesn't really matter I guess.

Thanks for fixing this so quickly

**Comment by simeonradivoev** [comment]
This is still not fixed on the latest version 1.1.4

**Comment by simeonradivoev** [comment]
For anyone having issues, make sure if your docker container was from an older version, you need to add the environmental variable `DB_ENGINE` to `django.db.backends.sqlite3`
